tests/test_basedmypy_typechecking.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import annotations
+
+from basedtyping import BASEDMYPY_TYPE_CHECKING
+
+if bool():  # noqa: UP018
+    # hide this from pytest
```

Comment:
whats this

Reply:
don't want pytest to throw on collection

Reply:
maybe make it a function that never gets called then? `bool()` is confusing because its impossible to remember whether it returns `True` or `False`

Reply:
what are you, a type checker?